.travis.yml

```diff
@@ -14,6 +14,7 @@ env:
     - LANG: en_US.UTF-8
     - BASEPATH: "$TRAVIS_BUILD_DIR"
     - ROOTSYS: "$BASEPATH/root"
+    - secure: lHXkz6p4ZifmMwIAxSsCaIUsAl1iUYxbid4Q1CAdsCcyHty8UXUillrYH+jpk9yhk3aTZZezM4j1szDBsDdHAHi+lUT1rse+UFclFrID9MVcfNLEkL7CQTktQ6/IPt/vvCLEPGfi/u8qz2L2PcQ+73pyJfbA/GWMihU16c4yOwzz0mdzhs8UgiEQ4CyBqUGEROnnwK2M/4nfuIpPKwJMfSll3ebWiox2wZBSdpni0U7mN7IV63IrfPsjl0N2VBIFsog/528C8USIMqncL4ArotO+kSVZ1OKYgw/LZe7EtC+XrpQDJlQkgC6riQlDvS38sFSYE2ERawZ3AlfSIIMaDifi4Ag1G7N3gcMfjWHoecrZomyEtNKEdwZ+V1jH7DlNePl/pmSwU6i4Mvrw9a+MB+VfT81D2exax1aaC0aJMSNpPWCi5pf/X182yBwO/89oOYqtdTE7J5CaVzxBTjFk4pPPzDZgPyFPOyLMF8QtvtcnRP6DRe5p6Yt0Ztqc5KK9+J/xj8YrKVJoVPlIvVZfmkn5RdPL6f7fcnugb7vl52E0ig+qFjHMvQkQhahXl0gBWb6PYsEzUOcVXdVn917pcWBTjzw4y+x4qgygGsTYI9KxSJbM4cl3mqHsJOxgRdUzKegg2wB8TlMT+4fN1oOZOpi2Vp43Hb4EO59KOyMGGmY=
```

Comment:
whats that?

Reply:
See https://docs.travis-ci.com/user/deployment/pypi/ (https://github.com/ComPWA/pycompwa/pull/87/commits/88391d1a56648d912a9a2fb7f524b7f830c35b57)

Reply:
Token has been renewed on PyPI